apps/web/pages/api/stripe/webhook.ts

```diff
@@ -1,7 +0,0 @@
-export const config = {
-  api: {
-    bodyParser: false,
-  },
-};
```

Comment:
we don't need this in app router because Route Handlers in App Router don't automatically parse the body